templates/onestop_english/templates.yaml

```diff
@@ -0,0 +1,78 @@
+dataset: onestop_english
+templates:
+  2807f792-45a6-4139-8386-7cdc98651e53: !Template
+    id: 2807f792-45a6-4139-8386-7cdc98651e53
+    jinja: 'For English as Second Language (ESL) learners, would the text passage
+      below be at {{"ele"}}, {{"int"}} or {{"adv"}} level for reading and simplifying?
+
+
+      "{{text}}"
+
+
+      |||
+
+
+      {{ ["ele", "int", "adv"] [label] }}'
+    name: esl_context
+    reference: ''
+    task_template: true
+  553f2bbd-269c-4c4f-bc12-3825f155844d: !Template
+    id: 553f2bbd-269c-4c4f-bc12-3825f155844d
+    jinja: 'Consider the following text passage: {{text}}
+
+
+      How would you rate the difficulty level of the passage above for automatic readability
+      assessment? {{"ele"}}, {{"int"}} or {{"adv"}} level?
+
+
+      |||
+
+
+      {{ ["ele", "int", "adv"] [label] }}'
```

Comment:
replace "ele" by "elementary", "int" by "intermediate", etc. in both the prompt and the label

Reply:
same for all templates